modules/release-notes/list-fixed-security-issues-in-product-1.3.2.txt

```diff
@@ -0,0 +1,8 @@
+# CVE number, affected package, fixed in version(s), JIRA
+
+# not yet fixed, built, or ready for release
+# CVE-2024-21534,jsonpath-plus,10.0.0,RHIDP-4440
+# CVE-2024-47762,@backstage/plugin-app-backend,0.3.75,RHIDP-4321
+# CVE-2024-45816,@backstage/plugin-techdocs-backend,1.10.13,RHIDP-4116
+# CVE-2024-46976,@backstage/plugin-techdocs-backend,1.10.13,RHIDP-4114
+# CVE-2024-45815,@backstage/plugin-catalog-backend,1.26.0,RHIDP-4118
```

Comment:
```suggestion
```
Remove all of these.  They will not be going into 1.3.x.  Requires a BS upgrade to get them